plugins/otter-pro/inc/plugins/class-patterns.php

```diff
@@ -90,6 +96,12 @@ public function register_patterns() {
 			return;
 		}
 
+		// Fast check to see if we have some corrupted data.
+		if ( ! $this->check_pattern_structure( $block_patterns[0] ) ) {
+			delete_transient( 'otter_pro_patterns' );
+			return;
+		}
```

Comment:
I'm bit worried about this. What happens if the user has corrupted data and the API continues to send back corrupted data? We will get into this loop of deleting and re-saving the transient. What do you think?

Reply:
I think we can put another transient for some time off?

Reply:
Sure, an hour sounds good. But we can do it only here in case where it's deleted on corrupt data, not when it doesn't exist.

Reply:
yes